build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -320,6 +352,14 @@ class SeleniumWebDriverController {
     return await handle.getElement().click();
   }
 
+  /**
+   * @return {!Promise<void>}
+   */
+  refresh() {
+    return this.driver.navigate().refresh()
+        .catch(e => {this.logError_(e);});
```

Comment:
Should remove curly braces here unless you plan on making this more than one line. This is also confusing since it looks like you are swallowing the error and not propagating it.